lib/ui/window/pointer_data_packet_converter_unittests.cc

```diff
@@ -129,6 +129,7 @@ void CreateSimulatedTrackpadGestureData(PointerData& data,  // NOLINT
   data.pan_delta_y = 0.0;
   data.scale = scale;
   data.rotation = rotation;
+  data.view_id = 0;
```

Comment:
Could we test the view ID packing/unpacking too?

Reply:
Done!